src/resources/elements/popupNotifications/popupNotifications.ts

```diff
@@ -182,6 +182,7 @@ export class PopupNotifications {
     };
 
     this.queueEventConfig(bannerConfig);
+    console.error(bannerConfig.message);
```

Comment:
@dkent600 
Thoughts?
Was helpful to get the console log to more quickly find the place in the code